docs/sources/administration/jaeger-instrumentation.md

```diff
@@ -0,0 +1,19 @@
++++
+title = "Jaeger instrumentation"
+description = "Jaeger traces emitted and propagation by Grafana"
+keywords = ["grafana", "jaeger", "tracing"]
+type = "docs"
+[menu.docs]
+parent = "admin"
+weight = 9
++++
+
+# Jaeger instrumentation
```

Comment:
When we get a better handle on the final shape of the documentation, we should revisit this title. I suspect we can come up with something more descriptive and accurate.